.github/workflows/comment.yml

```diff
@@ -58,7 +58,14 @@ jobs:
               .map(annotation => `${annotation.run}: ${annotation.annotation.message}`)
               .join('\n');
 
-            console.log(annotationContent);
+            // check if no changes
+            let gnuTestReport = annotationContent.includes('Run GNU findutils tests: Changes from main: PASS +0 / FAIL +0 / ERROR +0 / SKIP +0');
```

Comment:
this seems a bit fragile. In the json, we don't have the number to do some if > 0 ?

Reply:
That might mean getting the results for the master branch in comment.yml and comparing them to bfs-result.json and gnu-result.json? I think this might be a bit repetitive and complicated.

How about changing compare_bfs_result.py to send a Github Actions Annotations using `print(f"::warning :: BFS tests no change.")`  and catch annotation in `comment.yml` when compatibility is unchanged?

Reply:
yeah, probably better and easier